doc/api/package_json.md

```diff
@@ -0,0 +1,648 @@
+# Modules: Package configuration
```

Comment:
```suggestion
# Modules: The `package.json` file
```

Reply:
I'd like to have input from other collaborators on this; that's actually the approach I had taken on #32970, but @GeoffreyBooth had some outstanding objections and it never get through.